### PR TITLE
Condense provider, region and zone

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
The Infrastructure card takes up a lot of space. As an effort to reduce unnecessary rows, the provider, region and zone are now combined in one row.
Screenshot 2020-04-23 at 11 14 30

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user

```

### CODE REVIEW

File: frontend/src/components/Vendor.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<template>
+  <v-tooltip top>
```
Comment (holgerkoser) [inline frontend/src/components/Vendor.vue:18]: What is the value of the tooltip. It shows the same information in a different way.

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue [hunk 2]
```diff
@@ -25,9 +25,9 @@ limitations under the License.
           <v-icon color="cyan darken-2">cloud_queue</v-icon>
         </v-list-item-icon>
         <v-list-item-content>
-          <v-list-item-subtitle>Provider</v-list-item-subtitle>
+          <v-list-item-subtitle>{{vendorTitle}}</v-list-item-subtitle>
           <v-list-item-title class="pt-1">
-            {{shootCloudProviderKind}}
+            <vendor :shootItem="shootItem" :extended="true"></vendor>
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/ShootInfrastructureCard.vue:30]: The vendorTitle and the vendor a coupled. Not things are separated into different components. I would keep one thing in one component.

Comment (petersutter) [reply frontend/src/components/ShootDetails/ShootInfrastructureCard.vue:30]: hm, the vender component is also used in the shoot list row, so I can't put
v-list-item-subtitle and v-list-item-title into the vendor component, or are you thinking of a different solution?

File: frontend/src/components/Vendor.vue (hunk 1 above)
Comment (petersutter) [reply frontend/src/components/Vendor.vue:18]: yes on the details page it does not give any benefit, however on the shoot list row it does add some value
we could disable the tooltip on the shoot details page. WDYT?

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue (hunk 2 above)
Comment (holgerkoser) [reply frontend/src/components/ShootDetails/ShootInfrastructureCard.vue:30]: I was not aware of the fact that the vendor component is used in the ShootListRow. We could add a boolean prop `title` and if true it renders the title
```js
      if (this.shootZones.length > 0) {  
         return `Provider / Region / ${this.zoneTitle}`
      }
      return 'Provider / Region'
```

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue [hunk 3]
```diff
@@ -25,9 +25,9 @@ limitations under the License.
           <v-icon color="cyan darken-2">cloud_queue</v-icon>
         </v-list-item-icon>
         <v-list-item-content>
-          <v-list-item-subtitle>Provider</v-list-item-subtitle>
+          <v-list-item-subtitle>{{vendorTitle}}</v-list-item-subtitle>
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/ShootInfrastructureCard.vue:28]:
```suggestion
          <v-list-item-subtitle>
            <vendor title extended :shootItem="shootItem">
          </v-list-item-subtitle>
          <v-list-item-title class="pt-1">
            <vendor extended :shootItem="shootItem">
          </v-list-item-title>
```

File: frontend/src/components/Vendor.vue (hunk 1 above)
Comment (holgerkoser) [reply frontend/src/components/Vendor.vue:18]: see above

Comment (petersutter) [reply frontend/src/components/Vendor.vue]: ✅

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue (hunk 2 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/ShootInfrastructureCard.vue]: ✅

File: frontend/src/components/ShootDetails/ShootInfrastructureCard.vue (hunk 3 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/ShootInfrastructureCard.vue]: ✅ (did not accept suggestion here as the closing tag was missing, however I changed the code like you suggested)

**Pull request merged**